Wikipedia/UI-V5/WMFArticleFetcher.m

```diff
@@ -20,37 +188,45 @@ @implementation WMFArticleFetcher
 - (instancetype)initWithDataStore:(MWKDataStore*)dataStore {
     self = [super init];
     if (self) {
-        self.dataStore = dataStore;
-        AFHTTPRequestOperationManager* manager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
-        manager.responseSerializer = [AFHTTPResponseSerializer serializer];
-        self.operationManager      = manager;
+        self.operationManager.requestSerializer  = [WMFArticleRequestSerializer serializer];
+        self.operationManager.responseSerializer = [WMFArticleResponseSerializer serializer];
+        self.dataStore                           = dataStore;
     }
     return self;
 }
 
-- (ArticleFetcher*)fetcher {
-    if (!_fetcher) {
-        _fetcher = [[ArticleFetcher alloc] init];
-    }
+- (id)serializedArticleWithTitle:(MWKTitle*)title response:(NSDictionary*)response {
+    MWKArticle* article = [self.dataStore articleWithTitle:title];
```

Comment:
why are we reading the article from the data store? this seems to go against what the intent is here: to serialize `response` to the data store as the new state. i.e. we should only be writing in the fetcher, not reading (and even then, we should probably take the data store out of the fetcher entirely, and leave that concern to a higher-level controller object)